vcd/resource_vcd_catalog_test.go

```diff
@@ -256,6 +257,43 @@ func TestAccVcdCatalogPublishedToExternalOrg(t *testing.T) {
 		"PreserveIdentityInformationUpdate2": false,
 	}
 
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	// TODO - This code snippet is to avoid having the org catalog publishing settings set to disable.
+	// There are some bugs in VCD that disable those options. This code snippet will be removed
+	// as soon as those bugs are solved.
+	vcdClient := createTemporaryVCDConnection(false)
```

Comment:
You can make this work even when run in Org user mode by using `createSystemTemporaryVCDConnection` instead of `createTemporaryVCDConnection`.

Reply:
@Didainius changed and tested. It works with both sys admin and org admin